### PR TITLE
DIG-1686: Update query to work with updated katsu endpoints

### PR DESCRIPTION
**Pull request by OrdiNeu**
[Jira ticket](https://candig.atlassian.net/browse/DIG-1686)

# Description
This addresses changes from the [Katsu Model 3](https://github.com/CanDIG/katsu/pull/239) PR

# To Test:
1. Check out this branch, CanDIGv2-ingest's `model_3` [branch](https://github.com/CanDIG/candigv2-ingest/compare/develop...model_3), and Katsu's `model_3` [branch](https://github.com/CanDIG/katsu/tree/model_3)
2. Run Katsu ingest with the following:
```
conda activate candig

# Be in the root CanDIG directory for the following
python settings.py
source env.sh
cp env.sh lib/candig-ingest/candigv2-ingest/
cd lib/candig-ingest/candigv2-ingest/

export KATSU_URL=$CANDIG_URL/katsu

export CLINICAL_DATA_LOCATION=/home/fnguyen/Documents/CanDIGv2/lib/candig-ingest/candigv2-ingest/tests/clinical_ingest.json
python katsu_ingest.py
```
3. Modify candig-data-portal with the following: search and replace all instances of `v2/` with `v3/` except for the ones in `package-lock.json`, the Footer's `index.js`, the random line in the `.svg` file, and the ones that are /beacon/v2/g_variants`.
4. Load up the front page

### CODE REVIEW

File: query_server/query_operations.py [hunk 1]
```diff
@@ -75,22 +77,22 @@ def get_summary_stats(donors, headers):
...
 
-        # Cancer types
-        if donor['primary_site']:
-            for cancer_type in donor['primary_site']:
-                if cancer_type in primary_site_count:
-                    primary_site_count[cancer_type] += 1
-                else:
-                    primary_site_count[cancer_type] = 1
+        # primary sites
+        for primary_diagnosis in donor['primary_diagnosis']:
```
Comment (CourtneyGosselin) [inline query_server/query_operations.py]: Getting `KeyError: 'primary_diagnosis` resulting in a 500 error on the portal

**Review by CourtneyGosselin — CHANGES_REQUESTED** [review]
Along with the /query call that occurs when you load the clinical and genomic search page getting a 500 for key error on primary_diagnosis. There is also some 400 errors when I try and do some filtering

File: query_server/query_operations.py (hunk 1 above)
Comment (OrdiNeu) [reply query_server/query_operations.py]: Fixed in [561854c](https://github.com/CanDIG/candigv2-query/pull/45/commits/561854c117cb2a9d134a1a7a6184d26b1444b5b9)

**Review by CourtneyGosselin — APPROVED** [review]
LGTM works now with the data-portal PR

**Pull request merged**